### PR TITLE
Update ordering of information on course detail pages

### PR DESCRIPTION
**Pull request by TobyRet**
### Context
```
As a... user looking for teacher training courses
I need to... find out information about a course
So that... I know if I want to apply to it or not
```

### Changes proposed in this pull request
- Change ordering of sections

### Trello card
https://trello.com/c/26UU3jvU/2934-%F0%9F%97%BA-dev-update-ordering-of-information-on-course-detail-pages

### Checklist

- [x] Rebased master
- [x] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

File: app/views/courses/_about_schools.html.erb [hunk 1]
```diff
@@ -1,5 +1,13 @@
 <div class="govuk-!-margin-bottom-8">
   <h2 class="govuk-heading-l" id="section-schools"><%= course.placements_heading %></h2>
+  <aside class="app-advice" role="complementary">
+    <p class="app-advice__caption">Advice from Get into teaching</p>
+    <h3 class="app-advice__title">Where you will train</h3>
+    <div class="app-advice__body">
+      <p class="govuk-body">This course has placement schools. Most of your time will be spent in the classroom with experienced teachers.</p>
```
Comment (TobyRet) [inline app/views/courses/_about_schools.html.erb:7]: Travel to work information is not yet available. This means we cannot add the travel area to the 'where you will train' Get into Teaching Aside. We'll need a ticket to add this in when ready.

File: app/views/courses/_advice.html.erb [hunk 2]
```diff
@@ -1,8 +1,10 @@
-<h3 class="govuk-heading-l" id="section-advice">Support and advice</h3>
-<p class="govuk-body">For questions about this course you should contact the training provider using <%= link_to "the contact details above", "#contact_section", class: "govuk-link" %>.</p>
+<div class="govuk-!-margin-bottom-8">
+  <h3 class="govuk-heading-l" id="section-advice">Support and advice</h3>
+  <p class="govuk-body">For questions about this course you should contact the training provider using <%= link_to "the contact details above", "#contact_section", class: "govuk-link" %>.</p>
```
Comment (davidgisbey) [inline app/views/courses/_advice.html.erb:3]: Can we govuk_link for the links please 🙏

**Comment by paulrobertlloyd** [comment]
* `Entry requirements > Qualifications` has been renamed `Entry requirements >Qualifications needed` (but I know @george-bit has changes this in other PR so maybe can ignore)
* `Support and advice > Get support and advice about teaching` had been renamed to `Support and advice > For advice about teaching`

Perhaps in a separate PR (that we deploy) can we also update the following in the support section:

```diff
- from 8am to 8pm, Monday to Friday.
+ from 8.30am to 5pm, Monday to Friday (except public holidays).
```

File: app/views/courses/_advice.html.erb [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
 
-<h4 class="govuk-heading-m">Get support and advice about teaching</h4>
-<p class="govuk-body">Register with <%= link_to 'Get Into Teaching', 'https://beta-getintoteaching.education.gov.uk/helping-you-become-a-teacher', class: "govuk-link" %>, the Department for Education’s free support and advice service, for personalised guidance from teaching experts. They can help you to prepare your application, book school experience, and access exclusive teaching events. You can also call them free of charge on 0800 389 2500, or speak to an adviser using their <%= link_to "online chat service", "https://beta-getintoteaching.education.gov.uk/#talk-to-us", class: 'govuk-link' %>, from 8am to 8pm, Monday to Friday.</p>
+  <h4 class="govuk-heading-m">Get support and advice about teaching</h4>
```
Comment (davidgisbey) [inline app/views/courses/_advice.html.erb:5]: on the prototype this says `For advice about teaching`

**Review by davidgisbey — APPROVED** [review]
looks good 👍 feel free to ignore that comment about govuk_links if you want and I'll add a card to tech debt

**Pull request merged**